data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -183,19 +183,31 @@ podman run -d --name ironic-inspector-ramdisk-logs \
      --entrypoint /bin/runlogwatch.sh \
      -v $IRONIC_SHARED_VOLUME:/shared:z "${IRONIC_INSPECTOR_IMAGE}"
 
-# Now loop so the service remains active and restart everything should one of the containers exit unexpectedly.
-# The alternative would be RemainAfterExit=yes but then we lose the ability to restart if something crashes.
```

Comment:
Are we Ok with losing the restart if any container exits with a transient error?

I guess that only really helps if some initial error happens before terraform starts deploying via ironic-api, but I seem to remember adding that logic due to some occasional flaky failures with container startup...

Reply:
I added `--restart on-failure` to the podman commands, so I think that restores similar functionality, what do you think?

Reply:
Yeah I guess that should handle the majority of flaky-startup issues, and it avoids restarting all of them if there's a transient failure on only one.